config/README.rst

```diff
@@ -357,6 +357,11 @@ testenv-commands
   Replacement for the default ``commands`` option in ``[testenv]`` of
   ``tox.ini``. This option has to be a list of strings without indentation.
 
+testenv-deps
+  Replacement for the default ``deps`` option in ``[testenv]`` of ``tox.ini``.
+  This option has to be a list of strings without indentation.  The default is
+  ``['zope.testrunner']``.
```

Comment:
Most of the options are used in the example above, would it be an idea to have this for this option, too?